modules/components/src/ThemeSwitcher/StyleProvider.js

```diff
@@ -1,40 +1,39 @@
 import React from 'react';
 
 export default class StyleProvider extends React.Component {
-
   state = {
```

Comment:
props.page needs to be fed into state, we'll need a constructor here

Reply:
hmm not sure I understand. `props.page` isn't being used anywhere here?

Reply:
we could just hardcode it to be the 1, but the react-table does pass it in

Reply:
It could be a custom prop from ReactTable being passed in. Regardless, we need to initialize `state.page` to a number.